cockatrice/src/messagelogwidget.cpp

```diff
@@ -614,7 +614,9 @@ void MessageLogWidget::logSetCounter(Player *player, QString counterName, int va
         str = tr("%1 sets counter %2 to %3 (%4%5).", "female");
     else
         str = tr("%1 sets counter %2 to %3 (%4%5).", "male");
-    appendHtmlServerMessage(str.arg(sanitizeHtml(player->getName())).arg(QString("<font color=\"blue\">%1</font>").arg(sanitizeHtml(counterName))).arg(QString("<font color=\"blue\">%1</font>").arg(value)).arg(value > oldValue ? "+" : "").arg(value - oldValue));
+    QByteArray ba = counterName.toLatin1();
```

Comment:
Can't pass in a QString. The type wanted by tr is const char *. I mentioned it in the pr description.